src/category_theory/limits/small_complete.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Bhavik Mehta
+-/
+
+import category_theory.limits.shapes.products
+import set_theory.cardinal
+
+/-!
+# Any small complete category is a preorder
```

Comment:
This title suggests that there will be a `def` that gives you a term of type `preorder C` at some point.
Do you want to add that, or would you rather rephrase this doc to talk about thin categories.

Reply:
I explained in the doc what the actual statement is, but I hoped that in the title I could be a bit more vague and give the informal mathematical slogan; I don't mind changing it though. The difference is just that in Lean, a preorder category is one where the morphisms are in Prop, which is weaker than the usual notion of a preorder/thin category which says that each homset is subsingleton.

Reply:
I don't mind too much. If you add a sentence to the docs where you explain this explicitly, and state that therefore you don't provide a `preorder C` def, that's fine with me.

Reply:
Done!